doc/deployment/deploy_custom.md

```diff
@@ -53,7 +175,9 @@ You may either deploy manifests you created above or edit them to customize them
 
 ### Editing your manifest to customize it further
 
-This should only be approached if you are an experienced Kubernetes administrator. 
+This should only be approached if you are an experienced Kubernetes administrator.
+If you are attempting a highly customized deployment, 
+please engage with your Pachyderm support team. 
```

Comment:
Add slack link?

Reply:
That's in the footer of every page of documentation (which you don't see here, but will see when the doc is finally rendered on RTD).  If it's stated here, as well, it sounds clumsy, as it'll be placed above the exact same boilerplate.  

Of course, that *isn't* the case on github: you won't see the footer that RTD produces there.

@svekars, how should we handle this kind of thing?